src/content/docs/guides/hacktoberfest.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Hacktoberfest
+description: Learn how to contribute to Astro during Hacktoberfest.
+---
+
+[Hacktoberfest](https://hacktoberfest.com/about/) is an annual event during the month of October to encourage open-source contributions. If you've ever wanted to contribute to Astro, this is your opportunity. From beginner to experienced, from developer to low/no-code contributor, you're welcome!
+
+:::note
+Since Hacktoberfest happens once a year during October, assume the information on this page to be only valid and up-to-date during October of each year.
+
+Things such as participating repos, rules, instructions, rewards, and so on may vary from year to year.
+:::
+
+## Join the fest
+
+To take part in Hacktoberfest, here's all you need to do:
+
+1. Register for the event on [Hacktoberfest's official website](https://hacktoberfest.com/).
+2. Make any [valid contributions](#contributions) following [Hacktoberfest's rules](https://hacktoberfest.com/participation/#pr-mr-details) during October.
+3. Let us know you're participating by requesting your PRs to be labeled as `hacktoberfest-accepted`.
+
+## Primer on open-source
+
+Especially for first-time contributors, open-source can be quite frightening -- but don't worry, we're here to help you succeed! Here are a few reliable resources you can read to get up to speed on the matter:
+
+- [What is open-source](https://www.digitalocean.com/community/tutorials/what-is-open-source)
+- [How to contribute to open-source](https://opensource.guide/how-to-contribute/)
+- [How to use Git](https://www.digitalocean.com/community/cheatsheets/how-to-use-git-a-reference-guide)
+- [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/overview/getting-started-with-github-desktop)
+
+## Contributions
```

Comment:
```suggestion
## Allowed contributions
```
Not sure what exactly to put here, but allowable, accepted, etc. this heading should state that it is telling what is or isn't going to be accepted